### ISSUE TITLE
context command line argument

### ISSUE THREAD

**Issue by hadmut** [hetznercloud/cli#806]
### TL;DR

as far as I can see and do understand the --help texts, changing the context (i.e. token and project) requires an intermediate hcloud context use .. statement. 

This is awkward and error prone, e.g. when runing different programs/scripts/apps using hcloud simulteously in different windows, this way influencing each other.

I'd like to have each command, such as e.g. hcloud server list, to haven an additional parameter --context= .. to allow choosing the context for this particular run. 

### Expected behavior

I'd like to have a command line parameter to select the context for every command. (and, e.g.  to define an aliases for something like  alias monday="hcloud --context=monday"

**Comment by apricote** [comment]
Today is your lucky day! This was released 2 weeks ago as [v1.44.0](https://github.com/hetznercloud/cli/releases/tag/v1.44.0).

You can then specify the active context either through the `--context=...` flag, or through the environment variable `HCLOUD_CONTEXT=...`.

Hope this solves your problem, feel free to respond here if you have any further questions.
